app/oxalis/opengraph/OpenGraphService.scala

```diff
@@ -0,0 +1,152 @@
+package oxalis.opengraph
```

Comment:
would it be a lot of effort to rename oxalis --> webknossos?

Reply:
Nope, I’ll do that in a separate PR. Truth be told, the package structure doesn’t make that much sense even with the renaming, but it would certainly be an easy improvement